CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 14.2.0
```

Comment:
What happened to release 14.1.0 in the changelog... 🤔

Reply:
I spotted that too... I think I missed adding it in https://github.com/alphagov/gds-sso/pull/173

I'll push another commit adding the missing entry.